storage/client_status_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2016 The Cockroach Authors.
```

Comment:
This falls under the blanket of "status", we use the status name in other packages; it seemed appropriate.